CMakeLists.txt

```diff
@@ -90,6 +90,44 @@ set(MODMESH_INCLUDE_DIR "${CMAKE_CURRENT_LIST_DIR}/cpp" CACHE INTERNAL "")
 
 include_directories(${MODMESH_INCLUDE_DIR})
 
+set(MODMESH_USE_PYSIDE False CACHE BOOL "Choose to build with pyside or not")
```

Comment:
By default, turn off `-DMODMESH_USE_PYSIDE` because testing on Ubuntu and Windows is not yet done.